net/tac_plus/Makefile

```diff
@@ -25,27 +25,71 @@ CONFIGURE_ARGS += \
 	--disable-shared \
 	--without-libwrap
 
-define Package/tac_plus
+ifeq ($(BUILD_VARIANT),without-pam)
+CONFIGURE_VARS += \
+	ac_cv_lib_pam_pam_start=no
+endif
+
+define Package/tac_plus/Default
 	SECTION:=net
 	CATEGORY:=Network
```

Comment:
Would it make sense to put this under wireless?

Reply:
I think the Network category is more appropriate because this is a generic AAA service that can be used for more than just wireless (e.g. AAA on routers, switches, etc.).